In [2]:
import pyautogui
pyautogui.size()

Size(width=1920, height=1080)

* top_left = x=0 , y=0
* top_right= x=1920 , y=0
* bottom_left= x=0 ,y=1080
* bottom_right= x=1920 , y=1080

In [4]:
pyautogui.position()

Point(x=475, y=79)

In [3]:
pyautogui.position()
pyautogui.moveTo(x=819 , y=13, duration=2)
pyautogui.click()

pyautogui.moveTo(x=231 , y=75 , duration=2)
pyautogui.click()
pyautogui.write("youtube.com")
pyautogui.hotkey("enter")

pyautogui.moveTo(x=515 , y=145 , duration=2)
pyautogui.click(x=515 , y=145 , duration=1.5)
pyautogui.write("It expert by sir qasim")
pyautogui.hotkey("enter")

pyautogui.moveTo(x=770 , y=330 , duration=2)
pyautogui.click()

pyautogui.moveTo(x=406 , y=862 , duration=2)
pyautogui.click()

pyautogui.moveTo(x=783 , y=913 , duration=2)
pyautogui.click(x=783 , y=913 , duration=1)

pyautogui.moveTo(x=261 , y=776 , duration=2)
pyautogui.click()

pyautogui.moveTo(x=783 , y=913 , duration=2)
pyautogui.click(x=783 , y=913 , duration=1)

pyautogui.moveTo(x=261 , y=776 , duration=2)
pyautogui.click()

pyautogui.moveTo(x=783 , y=913 , duration=2)
pyautogui.click(x=783 , y=913 , duration=1)



FailSafeException: PyAutoGUI fail-safe triggered from mouse moving to a corner of the screen. To disable this fail-safe, set pyautogui.FAILSAFE to False. DISABLING FAIL-SAFE IS NOT RECOMMENDED.